python/kfserving/kfserving/protocols/protocol.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict, Union
+import numpy as np
+
+#TODO Needs further discussion if this is appropriate for the initial protocols we support: Tensorflow Serving HTTP and Seldon HTTP
+class RequestHandler(object):
+
+    def validate_request(self,body):
```

Comment:
I'm a fan of single verb APIs unless absolutely necessary (i.e `validate`).
Thoughts?